python/ray/util/client/server/server.py

```diff
@@ -62,7 +62,8 @@ def __init__(self, ray_connect_handler: Callable):
         self.state_lock = threading.Lock()
         self.ray_connect_handler = ray_connect_handler
```

Comment:
Can you make sure all connect_handlers in the code also have the extra_kwargs signature?

Reply:
Ohh! Great catch! Will update :) 

I think I can update some of the other tests to **not use connect_handlers** as they can now pass the kwargs directly in :)